### DISCUSSION TITLE
How to generate const/enum of some api method parameter?

### DISCUSSION THREAD

**Discussion by sgf** [microsoft/win32metadata#1984] [Q&A] [answered]
I read the documentation but it didn't explain exactly how to do it.
This caused me a lot of inconvenience, I thought everything should be fully automatic.

DrawDibBegin last parameter wFlags
 https://learn.microsoft.com/en-us/windows/win32/api/vfw/nf-vfw-drawdibdraw#parameters

Otherwise, where should I find the parameter definitions I need? Do I have to copy them manually? But this would be too inefficient. I don't want to worry about too many details.

**Comment by AArnott** [comment, marked as answer]
CsWin32 only constructs enums that are in the metadata. I'll transfer this to the metadata repo for consideration.
